firmware/Makefile

```diff
@@ -131,6 +131,16 @@ clean::
 	rm -f $(1).h $(3)
 endef
 
-$(eval $(call GENERATE_CODE,coin_info,defs,coin_info.c))
-$(eval $(call GENERATE_CODE,nem_mosaics,defs/nem/nem_mosaics.json,nem_mosaics.c))
 $(eval $(call GENERATE_CODE,bl_data,../bootloader/bootloader.bin))
+
+define MAKO
```

Comment:
This looks overcomplicated. Since there is only one output file, it should be sufficient to use something to the effect of

```make
%: %.mako defs
    $(Q)$(PYTHON) ../vendor/trezor-common/tools/cointool.py render $<
```

Reply:
Indeed, good catch! I got confused by the original code. Also we could throw away `GENERATE_CODE` and write the one invocation explicitly.